src/pages/home/report/ReportFooter.js

```diff
@@ -57,53 +57,54 @@ const defaultProps = {
     shouldDisableCompose: false,
 };
 
-class ReportFooter extends React.Component {
+const ReportFooter = ({report, reportActions, isOffline, onSubmitComment, errors, pendingAction, shouldShowComposeInput, shouldDisableCompose, isSmallScreenWidth, isComposerFullSize}) => {
```

Comment:
I think we should go with `function ReportFooter` instead of `const ReportFooter`.